### ISSUE TITLE
macOS project is copying some addon files to my project folder

### ISSUE THREAD

**Issue by dimitre** [openframeworks/openFrameworks#8063]
The first time I open xcodeproject it creates a folder "include/assimp" and copy a file to my project folder "assimp.a"
can you please take a look at this @danoli3 ?

<img width="504" alt="Screenshot 2024-07-28 at 20 42 52" src="https://github.com/user-attachments/assets/ee859c7b-9538-4a76-bd6c-f11ff921066b">

**Comment by danoli3** [comment]
Yes Xcode is doing this for addons I believe its how its adding the files to deploy phase or something.

projectGenerator issue.
Also I am noticing in deploy oF/lib folder (where obj files etc .an are built) its copying the slice there for core stuff. Need to configure something I think. not a bit issue but needs fixing

**Comment by roymacdonald** [comment]
it is not a project generator issue because it happens when the project is compiled, not when it gets created by PG. 
It has to do with the use of the xcframeworks
